scripts/create_sql_lite_dump.py

```diff
@@ -0,0 +1,125 @@
+    "id": "id",
+    "isb_core_id": "isb_core_id",
+    "indexUpdatedTime": "isb_updated_time",
+    "sourceUpdatedTime": "source_updated_time",
+    "label": "label",
+    "searchText": "search_text",
+    "description": "sample_description",
+    "hasContextCategory": "context_categories",
+    "hasMaterialCategory": "material_categories",
+    "hasSpecimenCategory": "specimen_categories",
+    "keywords": "keywords",
+    "informalClassification": "informal_classification",
+    "producedBy_isb_core_id": "produced_by_isb_core_id",
+    "producedBy_label": "produced_by_label",
+    "producedBy_description": "produced_by_description",
+    "producedBy_hasFeatureOfInterest": "produced_by_feature_of_interest",
+    "producedBy_responsibility": "produced_by_responsibility",
+    "producedBy_resultTime": "produced_by_result_time",
+    "producedBy_samplingSite_description": "produced_by_sampling_site_description",
+    "producedBy_samplingSite_label": "produced_by_sampling_site_label",
+    "producedBy_samplingSite_location_elevationInMeters": "produced_by_sampling_site_elevation_in_meters",
+    "producedBy_samplingSite_location_ll": "produced_by_sampling_site_location_ll",
+    "producedBy_samplingSite_placeName": "produced_by_sampling_site_place_name",
+    "registrant": "registrant",
+    "samplingPurpose": "sampling_purpose",
+    "curation_label": "curation_label",
+    "curation_description": "curation_description",
+    "curation_accessConstraints": "curation_access_constraints",
+    "curation_location": "curation_location",
+    "curation_responsibility": "curation_responsibility",
+    "relatedResource_isb_core_id": "related_resources_isb_core_id",
+    "source": "source"
+}
+
+NUMERIC_COLUMNS = ["produced_by_sampling_site_elevation_in_meters"]
+
+
+# Filter out any placeholder values that shouldn't appear in the dump
+def _filtered_value(value):
+    if type(value) is str and value == Transformer.NOT_PROVIDED:
+        return ""
+    return value
+
+
+@click.command()
+@click.option(
+    "-d",
+    "--db_url",
+    default=None,
+    help="The SQLite database file URL for storage.  Doesn't need to exist beforehand.",
+)
+@click.option("-s", "--solr_url", default=None, help="Solr index URL")
+@click.option(
+    "-v",
+    "--verbosity",
+    default="DEBUG",
+    help="Specify logging level",
+    show_default=True,
+)
+@click.option(
+    "-H", "--heart_rate", is_flag=True, help="Show heartrate diagnostics on 9999"
+)
+@click.option(
+    "-a",
+    "--authority",
+    default=None,
+    help="Which authority to use when selecting and dumping the resolved_content",
+)
+@click_config_file.configuration_option(config_file_name="isb.cfg")
+@click.pass_context
+def main(ctx, db_url, solr_url, verbosity, heart_rate, authority):
+    isb_lib.core.things_main(ctx, db_url, solr_url, verbosity, heart_rate)
+    engine = create_engine(ctx.obj["db_url"], echo=False)
+    SQLModel.metadata.drop_all(engine, tables=[ISBCoreRecord.__table__])
```

Comment:
Drop the table every time…made sense for how we're using this and certainly made development easier.

Reply:
Also, explicitly specify the table so we don't pull in any of the other stuff (Things and related indexes).